### PR TITLE
[PT-BR] Add `Upgrading to lazer`

### PR DESCRIPTION
**Pull request by Utor220**
## Self-check 

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] Apply https://github.com/ppy/osu-wiki/pull/8576

### CODE REVIEW

**Comment by The-Last-Cookie** [comment]
Please don't forget to apply the changes from #8548.

**Review by Utor220 — COMMENTED** [review]
Applied changes from #8548

File: wiki/Help_centre/Upgrading_to_lazer/pt-br.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
+---
+tags:
+  - game client
+---
+
+# Atualizando para o lazer
+
+osu!(lazer) é a próxima principal atualização para o jogo. É o fruto de vários anos de trabalho nos bastidores para meticulosamente reimplementar o jogo.
+
+O objetivo final é que esta versão seja lançada como uma atualização que irá substituir completamente a versão estável existente do jogo, mas ainda há algum trabalho a ser feito antes que isso seja possível. No fim das contas, **os jogadores** irão ditar quando isso acontecerá e nós iremos continuar a dar suporte para a versão anterior até que os jogadores tenham migrado de versão.
```
Comment (LeoFLT) [inline wiki/Help_centre/Upgrading_to_lazer/pt-br.md:10]:
```suggestion
O objetivo final é que esta versão seja lançada como uma atualização que irá substituir completamente a versão estável existente do jogo, mas ainda há algum trabalho a ser feito antes que isso seja possível. No fim das contas, **os jogadores** irão ditar quando isso acontecerá e nós iremos continuar a dar suporte para a versão anterior até que os jogadores tenham migrado.
```
Leve pleonasmo

File: wiki/Help_centre/Upgrading_to_lazer/pt-br.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+| Mod de Ajuste de dificuldade | ![No][false] | ![Yes][true][^difficulty-adjust] |
+| Pre-definição de mods | ![No][false] | ![Yes][true] |
+| Configuração por mod | ![No][false] | ![Yes][true] |
+| Novos mods "divertidos" | ![No][false] | ![Yes][true] |
+| Normalização das cores de combo [^normalisation] | ![No][false] | ![Yes][true] |
+| Segure para HUD | ![No][false] | ![Yes][true][^hold-for-hud] |
+| Ajuste de Offset | ![Partial][partial][^offset-calibration-stable] | ![Yes][true][^offset-calibration-lazer] |
+| osu! sliders "snake" queo segurados | ![No][false] | ![Yes][true][^can-disable] |
+| osu! "note lock" amigável para o jogador | ![No][false] | ![Yes][true][^note-lock] |
+| colorização das notas baseado em timing no osu!mania  | ![No][false] | ![Yes][true] |
```
Comment (LeoFLT) [inline wiki/Help_centre/Upgrading_to_lazer/pt-br.md:55]:
```suggestion
| colorização das notas baseado em timing no osu!mania | ![No][false] | ![Yes][true] |
```
Extra space

**Review by LeoFLT — CHANGES_REQUESTED** [review]
lgtm otherwise

File: wiki/Help_centre/Upgrading_to_lazer/pt-br.md (hunk 1 above)
Comment (Utor220) [reply wiki/Help_centre/Upgrading_to_lazer/pt-br.md]: Pronto, já mudei isso!

File: wiki/Help_centre/Upgrading_to_lazer/pt-br.md (hunk 2 above)
Comment (Utor220) [reply wiki/Help_centre/Upgrading_to_lazer/pt-br.md]: Resolvido também!

**Review by LeoFLT — APPROVED** [review]

**Comment by The-Last-Cookie** [comment]
And there's another one: #8576

**Comment by LeoFLT** [comment]
Translation for the new bit from #8576:

*Aviso: Nós ainda estamos fazendo ajustes ao equilíbrio do jogo e mecânicas. Por enquanto, **scores feitas no lazer não devem ser consideradas permanentes**.*

**Pull request merged**